Lib/test/test_linecache.py

```diff
@@ -65,7 +66,7 @@ def test_getline(self):
 
     def test_getlines(self):
         lines = linecache.getlines(self.file_name)
-        self.assertEqual(lines, self.file_list)
+        self.assertEqual(lines, self.test_list)
```

Comment:
I think it would be less intrusif not to change this but just override `EmptyFile.test_getlines()`. That you would not have to change all tests.

Reply:
Done. FWIW, this is horribly misabstracted.